var/spack/repos/builtin/packages/py-jaxlib/package.py

```diff
@@ -50,21 +72,3 @@ def install(self, spec, prefix):
             pip(*args)
         remove_linked_tree(self.tmp_path)
         remove_linked_tree(self.buildtmp)
-
-    def patch(self):
```

Comment:
Any reason to drop the patch method? I think it helps to logically separate patching from installation. Unless the multi-build system refactor broke this somehow.

Reply:
the multi-build system has yielded some surprising behavior here - not sure I like this solution, but see all the references to "wrapped_package_object" perhaps @alalazo has some words of wisdom?